app/src/main/res/values/attrs.xml

```diff
@@ -17,50 +16,62 @@
 
 <resources>
 
-    <attr name="title" format="string"/>
-    <attr name="is_required" format="boolean"/>
-    <attr name="hint" format="string"/>
+    <attr name="title" format="string" />
+    <attr name="is_required" format="boolean" />
+    <attr name="hint" format="string" />
 
     <declare-styleable name="EditTextDateField">
-        <attr name="picker_icon" format="integer"/>
+        <attr name="picker_icon" format="integer" />
     </declare-styleable>
 
     <declare-styleable name="BaseEditTextFormField">
-        <attr name="input_type" format="string"/>
-        <attr name="regex" format="string"/>
-        <attr name="is_required"/>
-        <attr name="hint"/>
-        <attr name="max_lines" format="integer"/>
-        <attr name="min_lines" format="integer"/>
-        <attr name="background_alpha" format="integer"/>
-        <attr name="multiline" format="boolean"/>
+        <attr name="input_type" format="string" />
+        <attr name="regex" format="string" />
+        <attr name="is_required" />
+        <attr name="hint" />
+        <attr name="max_lines" format="integer" />
+        <attr name="min_lines" format="integer" />
+        <attr name="background_alpha" format="integer" />
+        <attr name="multiline" format="boolean" />
     </declare-styleable>
 
     <declare-styleable name="SpinnerFormField">
-        <attr name="hint"/>
-        <attr name="is_required"/>
+        <attr name="hint" />
+        <attr name="is_required" />
     </declare-styleable>
 
     <declare-styleable name="BaseRadioGroupFormField">
-        <attr name="title"/>
-        <attr name="space_between_items" format="integer"/>
-        <attr name="is_required"/>
+        <attr name="title" />
+        <attr name="space_between_items" format="integer" />
+        <attr name="is_required" />
     </declare-styleable>
 
     <declare-styleable name="BaseCheckboxFormField">
-        <attr name="title"/>
-        <attr name="is_required"/>
+        <attr name="title" />
+        <attr name="is_required" />
     </declare-styleable>
 
     <declare-styleable name="FileSelectorField">
-        <attr name="dialog_title" format="string"/>
-        <attr name="tap_button_text" format="string"/>
-        <attr name="tap_button_title" format="string"/>
-        <attr name="src_tap_button" format="reference"/>
+        <attr name="dialog_title" format="string" />
+        <attr name="tap_button_text" format="string" />
+        <attr name="tap_button_title" format="string" />
+        <attr name="src_tap_button" format="reference" />
     </declare-styleable>
 
     <declare-styleable name="CustomButton">
-        <attr name="type" format="string"/>
+        <attr name="type" format="string" />
+    </declare-styleable>
+    <attr name="textInputLayoutTheme" format="reference" />
+
+    <declare-styleable name="CollapsibleView">
```

Comment:
if I want to enter a string  **@String/my_text** with collapsibleShowFooterText, It will work?

Reply:
Yes